apps/examples/testcase/le_tc/kernel/itc_libc_semaphore.c

```diff
@@ -62,6 +62,6 @@ static void itc_libc_semaphore_getprotocol_n_no_sem_init(void)
 
 int itc_libc_semaphore_main(void)
 {
-	itc_libc_semaphore_getprotocol_n_no_sem_init();
+	itc_libc_semaphore_getprotocol_no_sem_init_neg();
```

Comment:
same as previous
```
u/itc_module_feature_function_n/p_details
```
This is what we use in the naming and automation tool parses.
Please let me know why you change the naming.